addons/web/static/src/views/fields/fields.scss

```diff
@@ -65,6 +65,9 @@
     &.o_input, .o_input, &.o_form_label {
         color: $danger;
     }
+    &:not(.o_form_label) {
+        --o-input-background-color: #{rgba($o-danger, .1)};
```

Comment:
Indeed thanks! It's a better approach and easier to adapt if we want to change the visual result later :D Care that this works well with the input-bg having some transparency, with plain color it will need to be adapted but since they are now next to each other it's easy to manage.